src/main/java/org/spongepowered/api/event/entity/player/PlayerEvent.java

```diff
@@ -38,18 +38,6 @@
      *
      * @return The {@link Player} involved
      */
-    Player getPlayer();
-
-    @Override
-    Player getHuman();
-
-    @Override
-    Player getLiving();
-
     @Override
     Player getEntity();
-
-    @Override
-    Player getUser();
```

Comment:
Yes but `getUser()` returns `User`, not `Player`. The whole point of having the overrides is so that you can return a specific type of `User`, in this case, `Player`.